TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/FTC-roadrunner-tuning-note

```diff
@@ -0,0 +1,8 @@
+ftc dashboard
```

Comment:
You shouldn't put notes literally in the code base. If you need them, put them in your phone or something. Personally, I would just read the docs each time.

Reply:
This is just in case of network problem and convenience, should I put it in discord/google drive instead?

Reply:
No that's fine, I understand. Sorry for getting back to you so late, I hope your having a great holiday!